libs/ledger-live-common/src/device-core/managerApi/use-cases/getProviderIdUseCase.ts

```diff
@@ -0,0 +1,23 @@
+import { DeviceInfoEntity } from "../entities/DeviceInfoEntity";
+
+export const PROVIDERS: Record<string, number> = {
```

Comment:
Yes it could 👍  do you see an advantage of making it an enum ?
I will add the default provider regardless, good point!

Reply:
actually using an enum breaks some tests because then you can do access stuff with an index like PROVIDERS[1] and then stuff like this behaves in an unwanted way:
```
const providerName = PROVIDERS[postDash] ? postDash : null;
```

cf. this test run https://github.com/LedgerHQ/ledger-live/actions/runs/7755719007/job/21151656301

Reply:
Well the only advantage I see of making it an enum is the implicit typing. 
I prefer using enum for readability and I was pretty sure that [this behaviour was the same for a const or an enum.](https://www.typescriptlang.org/play?ts=5.0.4#code/KYOwrgtgBACgSgeQGoEkAiBROBlKBvKAFwHsSoBeKAIkIEM6qoBfAKAGNiQBnQqADwpQADO05diAG2AA6CcQDmACgQAjAFbA2haQGtgATy6L4ydFmwBKADSxEqTDgDafALo2T989JIkLQA)

Nevertheless LGTM 👍

Reply:
it's not the same behaviour because enum[x] is a reverse mapping  (it maps from value to name) https://www.typescriptlang.org/docs/handbook/enums.html#reverse-mappings
[cf playground](https://www.typescriptlang.org/play?ts=5.0.4#code/KYOwrgtgBACgSgeQGoEkAiBROBlA+hgOQFUBZKAbwCgooATYAMwEMwAbAFygF4oBGAGmp0mAZ25QATIJoBjVmABG4gMzSoIgBYBLdjID2WkGJ4AWNcGDiArIIC+AbkqV9RzvGToseBACEAUuI0VDT0zGzsAFx8arSiUVJCcopRqkKaOvqGIlFmQhZRNpQOTi4inAAe4rzOekZ6rMAAdKx6AOYAFO6omDj4xCQA2uUAuvywiN1euL5+Q8MAlEA):


(I didn't know this playground by the way, nice one)